### ISSUE TITLE
Chord.get cannot parse `G6add9`

### ISSUE THREAD

**Issue by shawnbro** [tonaljs/tonal#378]
It can parse `G6` and `Gadd9` though.  Is there a preferred way to combine multiple extensions?

**Comment by danigb** [comment]
You're right. It's a bug. I'll fix it soon

**Comment by shawnbro** [comment]
I'm taking a look at  it gonna see if I can create a PR to fix.

**Comment by danigb** [comment]
Meanwhile you can use some of the aliases: `Chord.get("GM69")` or `Chord.get("G69")`

**Comment by shawnbro** [comment]
Thanks.  It's interesting it thinks 6 is the octave in this case.

**Comment by danigb** [comment]
Yes, in fact, I'm going to remove the support for note with octaves in `Chord.get` just because 1) it's very confusing 2) chord symbols doesn't specify note octave

**Comment by shawnbro** [comment]
Makese sense. Thank you

**Comment by danigb** [comment]
BTW, you can use `Chord.get(["C", "69"])` to skip the octave

**Comment by shawnbro** [comment]
Nice! yeah looks like that worked, i'm getting `'G sixth/ninth'` as expected
